db/migrate/20200505152413_create_exercises.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :exercises do |t|
       t.belongs_to :track, foreign_key: true, null: false
 
-      t.string :uuid, null: false
+      t.string :uuid, null: false, index: { unique: true }
```

Comment:
Be aware that this currently will block ETL, as exercises in different tracks share the same UUIDs.

It's the right thing to do for sure, but just will break for practice exercises.

Reply:
Want me to temporary make it such that it is UUID/track that is unique?

Reply:
No, I think it'll be a good way to see failures when we pull the practice exercises in.